src/functional/ffront/field_operator_ast.py

```diff
@@ -143,6 +143,12 @@ class Compare(Expr):
     right: Expr
 
 
+class IfExp(Expr):
+    test: Expr
+    body: Expr
+    orelse: Expr
```

Comment:
```suggestion
class TernaryExpr(Expr):
    condition: Expr
    true_expr: Expr
    false_expr: Expr
```
As long as the naming in Python AST makes sense, we inherit the naming. The ternary operator is a good example where I believe this is not the case. I would propose the following naming.